.circleci/config.yml

```
@@ -6,7 +6,7 @@ version: 2.1
 
 defaults: &defaults
   docker:
-    - image: circleci/android:api-29
+    - image: circleci/android:api-31
```

```suggestion
    - image: cimg/android:2021.12.2
```

See https://circleci.com/docs/2.0/circleci-images/

---

Thanks! Moving a little too quickly 😄